### PR TITLE
add field and seed info

### PR DESCRIPTION
**Pull request by louy755**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,28 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+ClientFile.destroy_all
```
Comment (ssharba) [inline db/seeds.rb]: can we use smarter way to seed data? create a hash of data and loop through it.

**Review by ssharba — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (louy755) [reply db/seeds.rb]: so just remove this line or create a hash  of date ?

Comment (ssharba) [reply db/seeds.rb]: create a loop to create your objects

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,15 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```
Comment (ssharba) [inline db/seeds.rb]: @louy755 create client and client_files in one loop.

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,15 @@
...
+hashes = [
+  { file_format: "ali_lol_12052017.txt", regex: "DHP_Grp", client_id: 1 },
```
Comment (ssharba) [inline db/seeds.rb]: where did you get this file name? please check the card.

Comment (louy755) [reply db/seeds.rb]: proceeding!

File: db/seeds.rb (hunk 2 above)
Comment (louy755) [reply db/seeds.rb]: proceeding!

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,26 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+client_hashes = [
```
Comment (ssharba) [inline db/seeds.rb:8]: one nested loop only please

**Review by ssharba — CHANGES_REQUESTED** [review]

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,38 @@
...
+      },
+      {
+        file_format: 'LACClaimsRecovery.MMDDYYYY',
+        regex: 'LACClaimsRecovery'
+      }
+    ]
+  }
+]
+Client.delete_all
+ClientFile.delete_all
```
Comment (ssharba) [inline db/seeds.rb:37]: can we add something to the client model that will delete all client_files when we delete client?

Comment (louy755) [reply db/seeds.rb:37]: sure!

**Review by ssharba — APPROVED** [review]

**Pull request merged**